test/test_axes.jl

```diff
@@ -35,6 +35,18 @@ end
 @testset "Axis limits" begin
     pl = plot(1:5, xlims=:symmetric, widen = false)
     @test Plots.xlims(pl) == (-5, 5)
+    
+    pl = plot(1:3)
+    @test Plots.xlims(pl) == Plots.widen(1,3)
+    
+    pl = plot(0.5:0.5:2.5, xlims=:round)
+    @test Plots.xlims(pl) == (1, 3)
```

Comment:
I did that wrong, sorry
```suggestion
    pl = plot([1.05,2.0,2.95], ylims=:round)
    @test Plots.ylims(pl) == (1, 3)
```

Reply:
Slipped by me as well!